forced-translation/README.md

```diff
@@ -0,0 +1,69 @@
+# Apply forced translation using Marian
+
+This repository aims to give a walkthrough on how to train an end-to-end pipeline, providing the needed scripts and training guidelines to be able to successfully apply forced translation using Marian, fulfilling this way the second milestone of the [Marian-CEF](https://github.com/marian-cef/marian-dev) EU project.
+
+This milestone aims to support users in improving translation quality by injecting translations from their own bilingual terminology dictionaries. To expand coverage and support morphologically rich languages, it also supports inflecting entries in the bilingual terminology dictionary.
+
+We followed the approach taken in the following paper:<br/>
+> T. Bergmanis and M. Pinnis.  *Facilitating terminology translation with target lemma annotations.*  InProceedings of the 16th Conference of the European Chapter of the Association for Computational Linguistics:  Main  Volume,  pages  3105–3111,  Online,  Apr.  2021.  Association  for  Computational Linguistics. (https://www.aclweb.org/anthology/2021.eacl-main.271).
+
+All information about the experiments that we ran with the code provided in this repository, which data to use to reproduce them and the results can be found in the [docs](docs/Experiments.md) folder.<br/>
+The details regarding the technique used to be able to do forced translation are detailed in the paper cited above.
+
+We will now proceed with a description of how to use the repo.
+
+## Install
+
+Download and compile [Marian-NMT](https://github.com/marian-cef/marian-dev). For now it is needed to install the linked marian-cef version of Marian to use all the capabilities related to input factors, but as soon as the code is merged to marian-dev the link will be updated. Once Marian is installed add the path to the marian executables folder in the `.env` file.
```

Comment:
One quick question: what are the compilation options for Marian here? I am trying to run your script.